assets/javascripts/kitten/components/cards/team-card.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component, Fragment } from 'react'
+import { TeamCardTitle } from 'kitten/components/cards/team-card/title'
+import { TeamCardImage } from 'kitten/components/cards/team-card/image'
+import { TeamCardIcons } from 'kitten/components/cards/team-card/icons'
+
+export class TeamCard extends Component {
+  static Image = TeamCardImage
+  static Title = TeamCardTitle
+  static Icons = TeamCardIcons
+
+  render() {
+    return <Fragment>{this.props.children}</Fragment>
```

Comment:
Tu peux directement retourner `this.props.children`.

Reply:
Nope impossible. T'es obligé d'avoir au moins une balise `<Fragment>` autour.

Reply:
Sur Field, ca semble fonctionner https://github.com/KissKissBankBank/kitten/pull/1397/files#diff-64922a553e27c464e6b736d188402d01R15.